AuthenticatorBridgeKit/AuthenticatorBridgeItemDataModel.swift

```diff
@@ -6,6 +6,12 @@ import Foundation
 public struct AuthenticatorBridgeItemDataModel: Codable, Equatable {
     // MARK: Properties
 
+    /// The domain of the Bitwarden account that owns this item. (e.g. https://vault.bitwarden.com)
+    public let bitwardenAccountDomain: String?
+
+    /// The email of the Bitwarden account that owns this item.
+    public let bitwardenAccountEmail: String?
+
```

Comment:
🤔 IMO it shouldn't be necessary to have the `bitwarden` prefix on these new properties. Having only the `account` prefix should be enough to know the properties are from the Bitwarden account and not from the item being shared.
🤔  Another idea would be to have a different Authenticator bridge data model for account data so we don't mix account properties with item properties and also to only send that data once, these two properties will be the same for all items shared in the account. But this would require a larger change.
What's your thoughts on this? Also cc: @matt-livefront @KatherineInCode

Reply:
`accountDomain` and `accountEmail` seem clear enough to me too.

I think for simplicity, flattening the data model and including the domain and email with each one makes sense to me. The size of this data seems manageable and it'll be nice to not have to merge data from multiple tables for these shared items.

Reply:
Sounds good - I'll add a commit with `accountDomain` and `accountEmail` and drop the `bitwarden` prefix. 👍